### PR TITLE
suspicious-pacakge: Don't use pkg install

### PR DESCRIPTION
**Pull request by cprecioso**
rev2: Now with more `unar`!

### CODE REVIEW

File: Casks/suspicious-package.rb [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
   license :gratis
 
-  pkg 'SuspiciousPackage.pkg'
+  depends_on :formula => 'unar'
 
-  uninstall :pkgutil => 'com.mothersruin.pkg.SuspiciousPackagePlugin'
-end
+  container :type => :generic_unar
+  qlplugin 'Suspicious Package.qlgenerator'
+end
```
Comment (vitorgalvao) [inline Casks/suspicious-package.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

Comment (cprecioso) [inline Casks/suspicious-package.rb]: done

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 1493dfa2f0664d8fd840f83781b4d392251eafe4 to keep commit history cleaner. Your contribution is still credited to you.

**Comment by diimdeep** [comment]
Can't install due to current unar 1.8.1 is broken in 10.11, need to wait for 1.9.1 https://github.com/Homebrew/homebrew/pull/44314

`
==> Executing: ["#<Pathname:/usr/local/bin/unar>", "-q", "-D", "-o", "/var/folders/pw/380cgj9s1w7gnhtv063qsn1r0000gn/T/d20151001-8988-1nh62xy", "--", "#<Pathname:/Library/Caches/Homebrew/suspicious-package-latest.xip>"]
==> Purging files for version latest of Cask suspicious-package
Error: Command failed to execute!

==> Failed command:
["#<Pathname:/usr/local/bin/unar>", "-q", "-D", "-o", "/var/folders/pw/380cgj9s1w7gnhtv063qsn1r0000gn/T/d20151001-8988-1nh62xy", "--", "#<Pathname:/Library/Caches/Homebrew/suspicious-package-latest.xip>"]

==> Output of failed command:
/Library/Caches/Homebrew/suspicious-package-latest.xip: 

==> Exit status of failed command:
#<Process::Status: pid 9053 SIGSEGV (signal 11)>
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/system_command.rb:56:in `_assert_success'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/system_command.rb:33:in `run'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/system_command.rb:39:in `run!'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/container/generic_unar.rb:15:in `block in extract'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/tmpdir.rb:88:in `mktmpdir'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/container/generic_unar.rb:14:in `extract'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/installer.rb:110:in `extract_primary_container'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/installer.rb:65:in `install'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/cli/install.rb:20:in `block in install_casks'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/cli/install.rb:17:in `each'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/cli/install.rb:17:in `install_casks'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/cli/install.rb:6:in `run'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/cli.rb:79:in `run_command'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/hbc/cli.rb:119:in `process'
/usr/local/Cellar/brew-cask/0.56.0/rubylib/brew-cask-cmd.rb:18:in `<main>'
`

**Pull request closed without merging**